### PR TITLE
Support MSI in helm chart

### PR DESCRIPTION
**Pull request by gossion**
expose MSI in helm chart

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -36,6 +36,8 @@ providers:
     clientKey:
     tenantId:
     subscriptionId:
+    enableManagedIdentity: false
+    managedIdentityID:
```
Comment (feiskyer) [inline helm/values.yaml:40]: Could we merge these two into one `managedIdentityID`?

Comment (gossion) [reply helm/values.yaml:39]: sure, removed the bool flag

**Review by feiskyer — COMMENTED** [review]
LGTM

**Comment by feiskyer** [comment]
@gossion could you squash the commits?

**Review by feiskyer — APPROVED** [review]

**Pull request merged**